File: server/models/transaction.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class Transaction extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      // define association here
    }
  }
  Transaction.init({
    polid:  DataTypes.INTEGER,
    billadvisor:  DataTypes.STRING,
    netflag:  DataTypes.STRING,
    // prem comm ov1 ov2 ov3 disc
    transType:  DataTypes.STRING,
    txtype2:  DataTypes.INTEGER,
    // 1  , -1
    subType:  DataTypes.INTEGER,
    totalamt: DataTypes.FLOAT,
    paidamt:  DataTypes.FLOAT,
    remainamt:  DataTypes.FLOAT,
    status:  DataTypes.STRING,
    mainaccountcode: DataTypes.STRING,
    agentCode: DataTypes.STRING,
    insurerCode: DataTypes.STRING,
    policyNo:  DataTypes.STRING,
    endoseNo:  DataTypes.STRING,
    recieptno:  DataTypes.STRING,
    seqno:  DataTypes.INTEGER,
    grossprem: DataTypes.FLOAT,
    duty:  DataTypes.FLOAT,
    tax:  DataTypes.FLOAT,
    totalprem:  DataTypes.FLOAT,
    commamt: DataTypes.FLOAT,
    commtaxamt:  DataTypes.FLOAT,
    ovamt:  DataTypes.FLOAT,
    ovtaxamt: DataTypes.FLOAT,
    rprefdate:   DataTypes.DATEONLY,
    dfrpreferno: DataTypes.STRING,
    "premin-rprefdate":  DataTypes.DATEONLY,
    "premin-rprefdate-dfrpreferno": DataTypes.STRING,
    "premout-rprefdate":   DataTypes.DATEONLY,
    "premout-rprefdate-dfrpreferno": DataTypes.STRING,
    dueDate: DataTypes.DATEONLY,
    groupStamentID: DataTypes.INTEGER, 
    
    
    
  }, {
    sequelize,
    modelName: 'Transaction',
    schema: 'static_data'
  });
  return Transaction;
};

File: report/amityReport/Models/Transaction.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace amityReport.Models
{
    public class Transaction
    {
        [Key]public int Id { get; set; }
         public string TransType { get; set; } = null!;
         public string transStatus { get; set; } = null!;
        public int subType { get; set; }
         public string InsurerCode { get; set; } = null!;
         public string agentGroupCode { get; set; } = null!;
         public string policyNo { get; set; } = null!;
         public string agentCode { get; set; } = null!;
         public float amount { get; set; }
         public float duty { get; set; }
         public float stamp { get; set; }
         public float total { get; set; }
         public DateTime payDate { get; set; }
         public string payNo { get; set; } = null!;
         public DateOnly invoiceDate { get; set; }
         public string invoiceNo { get; set; } = null!;
         public string endoseNo { get; set; } = null!;
         public int level { get; set; }
         public DateOnly dueDate { get; set; }
        public DateTime createdAt { get; set; }
         public DateTime updatedAt  { get; set; }
         public int groupSatmentID { get; set; }
        [Column("totalamt")] public float Totalamt { get; set; }

        [Column("paidamt")] public float Paidamt { get; set; }
        [Column("remainamt")] public float Remainamt { get; set; }
        [Column("status")] public string Status { get; set; } = null!;
        [Column("seqno")] public int Seqno { get; set; }
        [Column("commamt")] public float Commamt { get; set; }
        [Column("commtaxamt")] public float Commtaxamt { get; set; }
        [Column("ovamt")] public float Ovamt { get; set; }
        [Column("ovtaxamt")] public float Ovtaxamt { get; set; }
        [Column("rprefdate")] public DateOnly Rprefdate { get; set; } 

        [Column("premin-rprefdate")]public DateOnly Preminrprefdate { get; set; }
        
        [Column("premout-rprefdate")]public DateOnly Premoutrprefdate { get; set; } 
     
    }
}
